frontend/src/lib/components/PropertyKeyInfo.tsx

```diff
@@ -648,7 +648,7 @@ export function PropertyKeyInfo({
         return (
             <Typography.Text
                 ellipsis={ellipsis}
-                style={{ color: 'inherit', maxWidth: 400, ...style }}
+                style={{ color: 'inherit', maxWidth: '100%', ...style }}
```

Comment:
Could this have any side-effects somewhere?

Reply:
🤔 there are 73 usages of the property key info. So as much as it seems wrong to me that it cares about its maximum width... I'm definitely not tracking them all down to check them 🤣